tests/parser.cpp

```diff
@@ -1538,35 +1535,31 @@ TEST(Parser, watchpoint_probe)
        "  int: 1\n");
 
   test_parse_failure("watchpoint:1b:8:w { 1 }", R"(
-stdin:1:1-18: ERROR: Found trailing non-numeric characters
-Invalid function/address argument
+stdin:1:1-18: ERROR: Invalid function/address argument: invalid trailing bytes: 1b
```

Comment:
How much of a pain is it to change it to this? (And some of the others below)
```suggestion
stdin:1:1-18: ERROR: Invalid function/address argument: 1b. Invalid trailing bytes.
```

Reply:
I'd rather keep it consistent and just have "error: <number>" as much as possible.